keypoints/include/pcl/keypoints/impl/iss_3d.hpp

```diff
@@ -411,7 +407,7 @@ pcl::ISSKeypoint3D<PointInT, PointOutT, NormalT>::detectKeypoints (PointCloudOut
     }
   }
 
-  bool* feat_max = new bool [input_->size()];
```

Comment:
For some variables in this file, I don't understand why you chose to add `{}` at the end (`feat_max`, `prg_mem`, `prg_local_mem`, 2x`omp_mem`, `borders`, `edge_points`). They weren't initialized with memset before, and initialization also doesn't seem necessary

Reply:
I thought it would cost nothing to zero-initialize them, but per Godbolt, for this example, appending `{}` increases the instruction count by ~10 instructions. I'll revert them.